layouts/index.html

```diff
@@ -85,6 +85,9 @@ <h2 class="h3 mt-3 text-center">Get Involved:</h2>
         <p></p>Join our Slack community: <a
           href="https://join.slack.com/t/cloud-hypervisor/shared_invite/enQtNjY3MTE3MDkwNDQ4LWQ1MTA1ZDVmODkwMWQ1MTRhYzk4ZGNlN2UwNTI3ZmFlODU0OTcwOWZjMTkwZDExYWE3YjFmNzgzY2FmNDAyMjI">Invite</a>
         </p>
+        <p></p>Participate in our community activities: <a
```

Comment:
You close the paragraph you just opened..

Reply:
Good catch. The syntax now makes more sense. Also fixed a similar issue few lines above.